### PR TITLE
Edit medications

### PR DESCRIPTION
**Pull request by dsamojlenko**
# Description

Add the ability to edit medications once entered.

## Test instructions

- Add a few conditions
- Add some medications
- Try editing one!

### CODE REVIEW

File: config/routes.config.js [hunk 1]
```diff
@@ -35,6 +35,10 @@ function getRoutes() {
         name: 'add_medication',
         path: { en: '/medication/add', fr: '/medication/add' },
       },
+      {
+        name: 'edit_medication',
+        path: { en: '/medication/:id/edit', fr: '/medication/:id/edit' },
```
Comment (CalvinRodo) [inline config/routes.config.js]: Can we have a proper french path here or some sort of path that reminds us this is not final text?

File: routes/medication/medication.njk [hunk 2]
```diff
@@ -15,8 +15,11 @@
                     <p class="text-base">{{ medication.medication_start_date }} - {{ medication.medication_end_date }}</p>
                 </div>
                 <div class="w-1/2">
-                    <!-- <a href="/en/medication/{{ loop.index }}/edit">Edit condition</a><br> -->
-                    <form action="/en/medication/{{ loop.index }}/delete" method="POST">
+                    <a href="/en/medication/{{ loop.index }}/edit" class="text-sm">
+                        <svg class="w-4 h-4 inline fill-current mr-2" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"><path d="M17.561 2.439c-1.442-1.443-2.525-1.227-2.525-1.227L8.984 7.264 2.21 14.037 1.2 18.799l4.763-1.01 6.774-6.771 6.052-6.052c-.001 0 .216-1.083-1.228-2.527zM5.68 17.217l-1.624.35a3.71 3.71 0 0 0-.69-.932 3.742 3.742 0 0 0-.932-.691l.35-1.623.47-.469s.883.018 1.881 1.016c.997.996 1.016 1.881 1.016 1.881l-.471.468z"/></svg>
+                        Edit medication
```
Comment (CalvinRodo) [inline routes/medication/medication.njk]: can we add some hidden text after `Edit medication` with the name of the medication to give this link some context.

File: views/base.njk [hunk 3]
```diff
@@ -11,6 +11,7 @@
 {%- from 'medical-consent.njk' import medicalConsent as medicalConsent with context -%}
 {%- from 'patientName.njk' import patientName as patientName with context -%}
 {%- from 'condition-form.njk' import conditionForm as conditionForm with context -%}
+{%- from 'medication-form.njk' import medicationForm with context -%}
```
Comment (CalvinRodo) [inline views/base.njk:14]: can this not be added just to the medication.njk file?

**Review by CalvinRodo — CHANGES_REQUESTED** [review]

File: config/routes.config.js (hunk 1 above)
Comment (dsamojlenko) [reply config/routes.config.js]: done

File: routes/medication/medication.njk (hunk 2 above)
Comment (dsamojlenko) [reply routes/medication/medication.njk]: done

File: views/base.njk (hunk 3 above)
Comment (dsamojlenko) [reply views/base.njk:14]: Decided to just group all of our custom templates here ... seems more organized

**Review by CalvinRodo — APPROVED** [review]

**Pull request merged**